packages/gatsby-source-wordpress/README.md

```diff
@@ -86,6 +86,14 @@ The Authentication on Wordpress.com is not supported yet. This means that you wo
           user: 'your-htaccess-username',
           pass: 'your-htaccess-password',
           sendImmediately: false
+
+          // If hostingWPCOM is true then you will need 
+          // to communicate with wordpress.com api
+          // in order to do that you need to create an app
+          // at https://developer.wordpress.com/apps/
+          // then add the clientId and clientSecret here
+          clientId: '',
```

Comment:
I think prefixing  that will generate alot of confusion because the user and pass are still used for oauth.